### PR TITLE
Rename Sky to Flutter in a few more places

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: sky/packages/sky/lib/src/rendering/binding.dart [hunk 1]
```diff
@@ -32,7 +32,7 @@ class _PointerState {
```
Comment (Hixie) [inline sky/packages/sky/lib/src/rendering/binding.dart]: remove the TODO at the top of this file

File: sky/packages/sky/lib/src/rendering/binding.dart [hunk 2]
```diff
@@ -42,9 +42,9 @@ class BindingHitTestEntry extends HitTestEntry {
 
 /// The glue between the render tree and the sky engine
 // TODO(ianh): rename this class FlutterBinding
```
Comment (Hixie) [inline sky/packages/sky/lib/src/rendering/binding.dart]: remove this TODO

File: sky/packages/sky_engine/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-Sky Engine
-==========
+Flutter Engine
+==============
 
 This packages describes the dart:sky library, which is the interface between
```
Comment (Hixie) [inline sky/packages/sky_engine/README.md]: This package, not packages.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**